src/style.css

```diff
@@ -17,3 +17,7 @@
   border-right: 10px solid transparent;
   border-bottom: 35px solid red;
 }
+
+.animated-marker {
+  transition: all 4s;
```

Comment:
@pedramp , will you change this to 7? Or do you have reasons to believe 4s is a better choice?

Reply:
I think it should be even smaller.